website/events-listener/src/config.ts

```diff
@@ -2,9 +2,9 @@ import dotenv from 'dotenv';
 import { strict as assert } from 'assert';
 dotenv.config();
 
-const checkEnv = (env) => {
+const checkEnv = (env: unknown): string => {
   assert.notStrictEqual(env, undefined);
-  return env;
+  return env as never;
```

Comment:
Why it's never?

Reply:
It's not really a never, just my my attempt to get away with casting to any ;3
Will think of better typings.

Reply:
I guess `env` can only be a string